frontend/packages/pipelines-plugin/src/components/pipelineruns/list-page/PipelineRunList.tsx

```diff
@@ -25,7 +26,15 @@ export const PipelineRunList: React.FC<PipelineRunListProps> = (props) => {
     'pipelines',
   );
   const [taskRuns, taskRunsLoaded] = useTaskRuns(props.namespace);
-
+  const [TRtaskRuns, TRtaskRunsLoaded] = useTRTaskRuns(props.namespace);
```

Comment:
```suggestion
  const [resultTaskRuns, resultTaskRunsLoaded] = useTRTaskRuns(props.namespace);
```

Reply:
In the long term, it might be col to have this included in the useTaskRuns with all the merge code below, or?